src/GitTools.Core/GitTools.Core.csproj

```diff
@@ -19,36 +19,36 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\output\debug\NET45\GitTools.Core\</OutputPath>
+    <OutputPath>..\..\output\Debug\GitTools.Core\net45\</OutputPath>
```

Comment:
For a product yes, but this is a library. Single library with multiple build platforms. Can change nuspec so src=net45\gittools.core\gittools.core.dll dest=lib\net45?

Sent from my Windows Phone
________________________________
From: Geert van Horrik<mailto:notifications@github.com>
Sent: ‎11/‎09/‎2015 6:42 PM
To: GitTools/GitTools.Core<mailto:GitTools.Core@noreply.github.com>
Cc: Jake Ginnivan<mailto:jake@ginnivan.net>
Subject: Re: [GitTools.Core] Solution update (#15)


In src/GitTools.Core/GitTools.Core.csproj<https://github.com/GitTools/GitTools.Core/pull/15#discussion_r39260019>:

> @@ -19,36 +19,36 @@
>      <DebugSymbols>true</DebugSymbols>
>      <DebugType>full</DebugType>
>      <Optimize>false</Optimize>
> -    <OutputPath>..\..\output\debug\NET45\GitTools.Core\</OutputPath>
> +    <OutputPath>..\..\output\Debug\GitTools.Core\net45\</OutputPath>


I normally follow this structure:

[platform][product]

Since if you are building, you are build against a platform and are not interested in other stuff. But if this is what nuget requires...

—
Reply to this email directly or view it on GitHub<https://github.com/GitTools/GitTools.Core/pull/15/files#r39260019>.